### PR TITLE
Adds replaces to edit form and show page. Adds rake task for mapping

### LINKED ISSUE #562: work ID => handle mapping rake task
Generate a rake task that can query for a mapping from work ID to its handle. The resulting map will be used to configure nginx to rewrite urls that are pointing at old DSpace handle links.

### LINKED ISSUE #571: expose replaces field on work form and presenters
The field is added, but inaccessible or viewable

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #571 Fixes #562

### CODE REVIEW

File: lib/tasks/handle_dump.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+namespace :scholars_archive do
+  desc "Dump all handles associated with all objects in db"
+  task :handle_dump do
+    puts "Beginning building"
+    @hash = {}
+    response = ActiveFedora::SolrService.get("replaces_tesim:*", :rows => 10000)["response"]["docs"]
```
Comment (wickr) [inline lib/tasks/handle_dump.rake]: Rows needs to be higher than 10000, I'd probably do 100,000

File: app/views/hyrax/base/_attribute_rows.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
 <%= presenter.attribute_to_html(:language, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
 <%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
-
+<%= presenter.attribute_to_html(:replaces, render_as: :faceted) %>
```
Comment (wickr) [inline app/views/hyrax/base/_attribute_rows.html.erb]: Replaces is not faceted, external link is probably fine

Comment (straleyb) [inline app/views/hyrax/base/_attribute_rows.html.erb]: I didnt think the handles were going to be on the show page or the form. Is that right? If so, Im not so sure this change is totally necessary.

File: app/presenters/etd_presenter.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class ETDPresenter < Hyrax::WorkShowPresenter
```
Comment (revgum) [inline app/presenters/etd_presenter.rb]: The convention seems to be EtdPresenter, you should rename this and the references to it.

**Comment by straleyb** [comment]
Is it ok if I alpha sort the metadata in another PR? I have that going along with adding in the rest of the metadata from the map.

**Comment by revgum** [comment]
@straleyb ah hah, my mistake.. carry on with that other PR for alpha sorting

**Comment by straleyb** [comment]
Ok updated @revgum 👍

**Pull request merged**